### PR TITLE
Adding Javascript tests using testling.

### PR DESCRIPTION
**Pull request by alvestrand**
This gives the hope of doing a cross-browser test.

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// This is a basic test file for use with testling.
+// The test script language comes from tape.
+var test = require('tape');
+
+m = require('../adapter.js');
+
+test('Browser identified', function(t) {
+  t.plan(2);
+  t.ok(m.webrtcDetectedBrowser);
```
Comment (phoglund) [inline test/test.js]: This currently prints 

ok 1 (unnamed assert)
ok 2 (unnamed assert)

which isn't very helpful, so I suggest you add a description (you can apparently do that as the second argument to ok). For instance, 'adapter.js should detect a browser'.

Have you ensure the test fails, i.e. that webrtcDetectedBrowser is indeed false if we don't detect a browser?

**Comment by phoglund** [comment]
One comment

**Comment by alvestrand** [comment]
I made the messages a little simpler "adapter.js should" is kind of implicit, since adapter.js is what it tests.

I also checked that if I misspell the variable name, the test fails. Should be good enough to demonstrate that the framework does what it should.

**Comment by phoglund** [comment]
lgtm

**Pull request merged**